### PR TITLE
Team member page: final touchups

### PR DESCRIPTION
**Pull request by pzduniak**
@jakob223 I still have to make sure that the design works on mobile (up to date as of 21/03). I ended up running the useAllChannelsMeta hook in the list elements, but I could fire off the RPC in my "bulk fetch" too, if desired.

`ListItem2` didn't work due to the dynamic height, the padding jumped a lot and it was impossible to fix without introducing lots of additional CSS in the ListItem2 component. Since the behaviour did not match the mocks, I just reimplemented it from scratch.

cc @keybase/y2ksquad

### CODE REVIEW

File: shared/common-adapters/box.desktop.tsx [hunk 1]
```diff
@@ -83,7 +83,7 @@ const _Box2 = (props: Box2Props, ref: React.Ref<HTMLDivElement>) => {
       ]
         .filter(Boolean)
         .join(' ')}
-      style={(style as unknown) as React.CSSProperties}
+      style={(Styles.collapseStyles([style]) as unknown) as React.CSSProperties}
```
Comment (jakob223) [inline shared/common-adapters/box.desktop.tsx]: I thought we were trying to move toward calling `Styles.collapseStyles` before passing the style into components (cc @chrisnojima ?)

File: shared/common-adapters/dropdown.tsx [hunk 2]
```diff
@@ -119,15 +119,15 @@ type InlineDropdownProps = {
   containerStyle?: Styles.StylesCrossPlatform
   label: React.ReactNode
   onPress: () => void
-  type: 'Body' | 'BodySmall' | 'BodySmallSemibold'
+  type?: 'Body' | 'BodySmall' | 'BodySmallSemibold'
```
Comment (jakob223) [inline shared/common-adapters/dropdown.tsx]: hmm this seems possibly liable to make it easier for people to misuse this. WDYT of
```
type Props = { ... most of them ... } & 
({label: string; type: 'Body' | etc} | {label: React.ReactElement; type?: 'Body' | etc})
```

File: shared/teams/common/activity.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import * as Types from '../../constants/types/teams'
 
 const activityToIcon: {[key in 'active' | 'recently']: Kb.IconType} = {
   active: 'iconfont-fire',
-  recently: 'iconfont-team-leave',
+  recently: 'iconfont-typing',
```
Comment (jakob223) [inline shared/teams/common/activity.tsx]: are you sure you want `typing` and not `campfire-out`?

File: shared/teams/team/member/index.new.tsx [hunk 4]
```diff
@@ -170,43 +171,97 @@ const SubteamNotInRow = (props: SubteamNotInRowProps) => {
...
   const memberCount = props.subteam.memberCount ?? -1
-  const body = (
-    <Kb.Box2 direction="vertical" alignItems="flex-start">
-      <Kb.Text type="BodySemibold">{props.subteam.teamname}</Kb.Text>
-      <Kb.Text type="BodySmall">
-        {memberCount.toLocaleString()} {pluralize('member', memberCount)}
-      </Kb.Text>
+
+  return (
+    <Kb.Box2 direction="vertical" fullWidth={true} style={styles.rowCollapsedFixedHeight}>
```
Comment (jakob223) [inline shared/teams/team/member/index.new.tsx]: maybe add a comment here to replace this with a ListIem2 when we fix the padding issues on it

File: shared/teams/team/member/index.new.tsx [hunk 5]
```diff
@@ -244,49 +302,17 @@ const SubteamInRow = (props: SubteamInRowProps) => {
...
+  const loadingActivity = Container.useAnyWaiting(
+    Constants.loadSubteamActivityWaitingKey(props.subteam.id, props.username)
   )
-  const action = (
+
+  const channelsJoined = Array.from(channels)
+    .map(([_, {channelname}]) => channelname)
+    .join(', #')
+
+  const height = expanded ? (Styles.isMobile ? 208 : 140) : undefined
```
Comment (jakob223) [inline shared/teams/team/member/index.new.tsx]: this is unused according to TS

File: shared/util/feature-flags.desktop.tsx [hunk 6]
```diff
@@ -14,7 +14,7 @@ const ff: FeatureFlags = {
   newTeamBuildingForChatAllowMakeTeam: false,
   tabletSupport: true, // Whether tablet support is public. Changes some UI on non-tablets.
   teamInvites: false,
-  teamsRedesign: false,
+  teamsRedesign: true,
```
Comment (jakob223) [inline shared/util/feature-flags.desktop.tsx]: 🔥

File: shared/util/feature-flags.native.tsx [hunk 7]
```diff
@@ -14,7 +14,7 @@ const ff: FeatureFlags = {
   newTeamBuildingForChatAllowMakeTeam: false,
   tabletSupport: true,
   teamInvites: false,
-  teamsRedesign: false,
+  teamsRedesign: true,
```
Comment (jakob223) [inline shared/util/feature-flags.native.tsx]: 🚒

File: shared/common-adapters/box.desktop.tsx (hunk 1 above)
Comment (pzduniak) [reply shared/common-adapters/box.desktop.tsx]: i fixed another issue like this with nojima a week ago or so: https://github.com/keybase/client/pull/23025/files

File: shared/teams/common/activity.tsx (hunk 3 above)
Comment (pzduniak) [reply shared/teams/common/activity.tsx]: https://app.zeplin.io/project/5e2b67989d77855729413dce/screen/5e2b61487119da7e69fb64a9

you tell me :D

Comment (jakob223) [reply shared/teams/common/activity.tsx]: I think you want fire-out for this, and you want to use the typing icon in the place where you presumably already have it in the other screen

File: shared/teams/team/member/index.new.tsx (hunk 4 above)
Comment (pzduniak) [reply shared/teams/team/member/index.new.tsx]: I honestly don't know if we want to do it at this point. The expansion stuff seems impossible to properly implement in a LI2 :/

File: shared/common-adapters/dropdown.tsx (hunk 2 above)
Comment (pzduniak) [reply shared/common-adapters/dropdown.tsx]: Good call!

**Pull request merged**